src/css/index.css

```diff
@@ -28,11 +28,3 @@
  */
 @tailwind utilities;
 
-/**
- * Use this directive to control where Tailwind injects the responsive
- * variations of each utility.
- *
- * If omitted, Tailwind will append these classes to the very end of
- * your stylesheet by default.
- */
-@tailwind variants;
```

Comment:
Removing this one because we no longer need this line
https://tailwindcss.com/docs/installation